posthog/auth.py

```diff
@@ -89,7 +89,7 @@ def authenticate(cls, request: Union[HttpRequest, Request]) -> Optional[Tuple[An
             key_last_used_at.day,
             key_last_used_at.hour,
         ):
-            key_last_used_at = now
+            personal_api_key_object.last_used_at = now
             personal_api_key_object.save()
```

Comment:
```suggestion
            personal_api_key_object.save(update_fields=["last_used_at"])
```

Would also accept an `PersonalAPIKey.objects.update( ... `

It's not 100% obvious though that there might not be other values that should also be updated. I'll leave to you to decide.

Reply:
yeah `update()` is a lot dicier, will use save with specific fields.